### ISSUE TITLE
Default sorting option

### ISSUE THREAD

**Issue by Erebea** [floogulinc/hydrus-web#119]
Currently it seems "import time: newest first" is hardcoded as the default sorting method when browsing files. I would like it if this could be changed in the settings menu, as "modified time: newest first" is much more useful for me.

**Comment by McCarthee** [comment]
Yes, even if it just remembered the last one you chose would be a big upgrade.
